testing/yakuake/APKBUILD

```diff
@@ -0,0 +1,31 @@
+# Contributor: Bart Ribbers <bribbers@disroot.org>
+# Maintainer: Bart Ribbers <bribbers@disroot.org>
+pkgname=yakuake
+pkgver=3.0.5
+pkgrel=0
+arch="all"
+url="https://yakuake.kde.org/"
+pkgdesc="A drop-down terminal emulator based on KDE Konsole technology"
+license="GPL-2.0-only OR GPL-3.0-only"
+makedepends="extra-cmake-modules qt5-qtbase-dev qt5-qtsvg-dev qt5-qtx11extras-dev karchive-dev kconfig-dev kcoreaddons-dev kcrash-dev kdbusaddons-dev kglobalaccel-dev ki18n-dev kiconthemes-dev kio-dev knewstuff-dev knotifications-dev knotifyconfig-dev kparts-dev kwidgetsaddons-dev kwindowsystem-dev kwayland-dev"
+checkdepends="xvfb-run"
+source="https://download.kde.org/stable/$pkgname/$pkgver/src/$pkgname-$pkgver.tar.xz"
+subpackages="$pkgname-lang"
+
+build() {
+	cmake \
+		-DCMAKE_BUILD_TYPE=RelWithDebInfo \
+		-DCMAKE_INSTALL_PREFIX=/usr \
+		-DCMAKE_INSTALL_LIBDIR=lib
+	make
+}
+
+check() {
+	CTEST_OUTPUT_ON_FAILURE=TRUE ctest
```

Comment:
xvfb-run isn't used here (is it used implicitly ?)

Reply:
Sorry my mistake, checkdepends is not needed. Fixed it.